content/docs/1.11.0/best-practices.md

```diff
@@ -69,12 +69,12 @@ The following Linux OS distributions and versions have been verified during the
 | No. | OS                           | Versions
 |-----|------------------------------| --------
 | 1.  | Ubuntu                       | 24.04
-| 2.  | SUSE Linux Enterprise Server | 15 SP7
+| 2.  | SUSE Linux Enterprise Server | 16
 | 3.  | SUSE Linux Enterprise Micro  | 6.1
-| 4.  | Red Hat Enterprise Linux     | 10.0
-| 5.  | Oracle Linux                 | 9.6
-| 6.  | Rocky Linux                  | 10.0
-| 7.  | Talos Linux                  | 1.10.6
+| 4.  | Red Hat Enterprise Linux     | 10.1
+| 5.  | Oracle Linux                 | 10.0
+| 6.  | Rocky Linux                  | 10.1
+| 7.  | Talos Linux                  | 1.11.5
 | 8.  | Container-Optimized OS (GKE) | 121
```

Comment:
<img width="463" height="511" alt="Image" src="https://github.com/user-attachments/assets/9158fae0-f779-4b9b-be31-989fcd2e57e9" />

Ubuntu => 22.04 and 24.04
SUSE Linux Enterprise Server => 16 and 15 SP7
SUSE Linux Enterprise Micro => 6.0 and 6.1
Red Hat Enterprise Linux  => 10.0 and 10.1


cc @longhorn/qa